integrations/kubernetes-ingress/external-secrets.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: ngrok External Secrets integration
+description: ngrok integrates with External Secrets Operator to enable secure management of secrets used in ngrok Traffic Policy.
+sidebarTitle: External Secrets
+---
+
+This integration allows you to push Kubernetes secrets to ngrok vaults, where they can be referenced in your traffic policies for secure configuration management.
+
+## How it works
+
+[External Secrets Operator](https://external-secrets.io/) watches for PushSecret resources in your cluster.
+When a PushSecret is created or updated, it reads the specified Kubernetes secret and pushes the secret data to your ngrok vault using the ngrok API.
+The secret then becomes available in ngrok for use in Traffic Policies.
+The operator continues to sync changes based on the configured refresh interval, ensuring your ngrok secrets stay up-to-date with changes in Kubernetes.
+
+### Limitations
+
+- Currently supports push operations only (Kubernetes → ngrok)
+- Pull operations (ngrok → Kubernetes) are not yet supported
+
+## Use cases
+
+- **Traffic Policy configuration:** Store API keys, tokens, and credentials used in ngrok Traffic Policy rules
```

Comment:
```suggestion
- **Traffic Policy configuration:** Store API keys, tokens, and credentials that you can use to authenticate visitors with traffic policy actions
```
Only gotta capitalize the first "traffic policy" under the H2 (maybe this rule is more confusing than we initially thought 😅. I may have forgotten to correct the capitalization in a few places). 

I think we mean tp actions here, not rules?

Also, I added a little bit about auth to make the actual value add more concrete here